src/System.IO.FileSystem/tests/WinRT_BrokeredFunctions.cs

```diff
@@ -0,0 +1,207 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+using Xunit;
+using Windows.Storage;
+
+namespace System.IO.Tests
+{
+    [PlatformSpecific(TestPlatforms.Windows)]
+    [SkipOnTargetFramework(~(TargetFrameworkMonikers.Uap | TargetFrameworkMonikers.UapAot))]
+    public partial class WinRT_BrokeredFunctions : FileSystemTest
+    {
+        private static string s_musicFolder = StorageLibrary.GetLibraryAsync(KnownLibraryId.Music).AsTask().Result.SaveFolder.Path;
+
+        [Fact]
+        public void CopyFile_ToBrokeredLocation()
+        {
+            string testFile = GetTestFilePath();
+            File.Create(testFile).Dispose();
+
+            string destination = Path.Combine(s_musicFolder, "CopyToBrokeredLocation_" + Path.GetRandomFileName());
+            try
+            {
+                Assert.False(File.Exists(destination), "destination shouldn't exist before copying");
+                File.Copy(testFile, destination);
+                Assert.True(File.Exists(testFile), "testFile should exist after copying");
+                Assert.True(File.Exists(destination), "destination should exist after copying");
+            }
+            finally
+            {
+                File.Delete(testFile);
+                File.Delete(destination);
+            }
+        }
+
+        [Fact]
+        public void CreateDirectory()
+        {
+            string testFolder = Path.Combine(s_musicFolder, "CreateDirectory_" + Path.GetRandomFileName());
+            try
+            {
+                Assert.False(Directory.Exists(testFolder), "destination shouldn't exist");
+                Directory.CreateDirectory(testFolder);
+                Assert.True(Directory.Exists(testFolder), "destination should exist");
+            }
+            finally
+            {
+                Directory.Delete(testFolder);
+            }
+        }
+
+        [Fact]
+        public void DeleteFile()
+        {
+            string testFile = Path.Combine(s_musicFolder, "DeleteFile_" + Path.GetRandomFileName());
+            CreateFileInBrokeredLocation(testFile);
+            Assert.True(File.Exists(testFile), "testFile should exist before deleting");
+            File.Delete(testFile);
+            Assert.False(File.Exists(testFile), "testFile shouldn't exist after deleting");
+        }
+
+        [Fact]
+        public void FindFirstFile()
+        {
+            string subFolder = Path.Combine(s_musicFolder, "FindFirstFile_SubFolder_" + Path.GetRandomFileName());
+            Directory.CreateDirectory(subFolder);
+            string testFile = Path.Combine(subFolder, "FindFirstFile_SubFile_" + Path.GetRandomFileName());
+            CreateFileInBrokeredLocation(testFile);
```

Comment:
Long path is an issue here, unfortunately. They won't work in the brokered locations unless the Windows policy is enabled.  I'll tweak the file creation to dump some text identifying it as a CoreFX test file.